src/Components/Resource/ResourceDetails.tsx

```diff
@@ -103,10 +103,7 @@ export default function ResourceDetails(props: { id: string }) {
     return (
       <div id="section-to-print" className="print bg-white">
         <div className="flex justify-center">
-          <img
-            src={`${import.meta.env.REACT_PUBLIC_URL}/${KeralaLogo}`}
-            alt="kerala-logo"
-          />
+          <img src={`/${KeralaLogo}`} alt="kerala-logo" />
```

Comment:
Shouldn't this be replaced with the symbol of the state for which the instance is deployed?

Reply:
the `KeralaLogo` const is only being used here

Reply:
@mathew-alex could you comment? This is about the approval letter in the resource section.
![image](https://user-images.githubusercontent.com/25143503/231052234-4de521f8-1e65-464b-9d4a-440a9deba90d.png)

Reply:
After discussing it with @nihal467 it's decided to be removed for now. If we need to show `state_logo` it'll be followed up by #5318